cp4d/5.0.x/cp4d-cloud-pak-deployer-pipeline.yaml

```diff
@@ -1804,13 +1804,13 @@ spec:
                       state: $(params.llm-granite-13b-chat-v2-state)
                     - model_id: ibm-granite-13b-instruct-v2
                       state: $(params.llm-ibm-granite-13b-instruct-v2-state)
-                    - model_id: ibm-granite-3b-code-instruct
+                    - model_id: granite-3b-code-instruct
                       state: $(params.llm-ibm-granite-3b-code-instruct-state)
-                    - model_id: ibm-granite-8b-code-instruct
+                    - model_id: granite-8b-code-instruct
                       state: $(params.llm-ibm-granite-8b-code-instruct-state)
-                    - model_id: ibm-granite-20b-code-instruct
+                    - model_id: granite-20b-code-instruct
                       state: $(params.llm-ibm-granite-20b-code-instruct-state)
-                    - model_id: ibm-granite-34b-code-instruct
+                    - model_id: granite-34b-code-instruct
                       state: $(params.llm-ibm-granite-34b-code-instruct-state)
                     - model_id: ibm-granite-20b-multilingual
```

Comment:
Do different models (line 1815) not need the renaming? 

May be worth a comment here since YAML supports it.

Reply:
Yea this is just a random change that happened for all of the 5.0.1 models that I didn't realize and resulted in them breaking.

These models are copied directly from https://www.ibm.com/docs/en/cloud-paks/cp-data/5.0.x?topic=install-foundation-models but somehow I missed that these ones started breaking the pattern.

These values are actually something that the user will have no visibility of as they only interact with the params (llm-ibm-granite-34b-code-instruct-state) which I kept consistent. I'm considering changing the params though too but for now I'm following the old pattern.